### PR TITLE
fix: change data

### PR DESCRIPTION
**Pull request by WizardlyDev**

### CODE REVIEW

File: CREDITS.md
Comment (TyreseLian) [inline CREDITS.md:1]: good!

File: README.md
Comment (TyreseSong) [inline README.md:1]: good

File: package-lock.json
Comment (TyreseJin) [inline package-lock.json:1]: good!

File: src/redux/initialState.js
Comment (TyreseApollo) [inline src/redux/initialState.js:1]: good!

**Review by TyreseApollo — COMMENTED** [review]
good!

**Review by TyreseJin — COMMENTED** [review]
good!

**Review by TyreseSong — COMMENTED** [review]
good!

**Review by TyreseLian — COMMENTED** [review]
good!

**Comment by WizardlyDev** [comment]
good!

**Review by WizardlyDev — COMMENTED** [review]
good!

**Review by WizardlyDev — COMMENTED** [review]
good!

**Pull request merged**